packages/api/src/beacon/client/index.ts

```diff
@@ -42,5 +42,6 @@ export function getClient(opts: HttpClientOptions, modules: ClientModules): ApiC
     node: node.getClient(config, httpClient),
     proof: proof.getClient(config, httpClient),
     validator: validator.getClient(config, httpClient),
+    httpClient,
```

Comment:
Alternative to exposing the http client here would be to emit the warning inside the http client itself but that's not ideal either because a) it does not have any context to what it is connected so  log would have be more generic and b) it's does not have access to clock, could emit the warning at an arbitrary interval but it felt cleaner that way.